### PR TITLE
chore: track ibc-rs PR 1158

### PR DESCRIPTION
**Pull request by Farhad-Shabani**
Tracks: https://github.com/cosmos/ibc-rs/pull/1158

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1034,8 +1034,7 @@ dependencies = [
 [[package]]
 name = "ibc"
 version = "0.51.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "8af2325614ae3274b2e70834bea56c1fa2758d802d364d89992b0fe820847fca"
+source = "git+https://github.com/cosmos/ibc-rs.git#f7a3c2b23f42977392b2f2ca908cc98d00765ff5"
```
Comment (rnbguy) [inline Cargo.lock:1037]: looks like, the rev is set to wrong rev (it's the latest commit on `main`) :sweat_smile:

File: Cargo.toml [hunk 2]
```diff
@@ -46,4 +46,7 @@ tendermint-abci  = { version = "0.34", default-features = false }
 tendermint-proto = { version = "0.34", default-features = false }
 tendermint-rpc   = { version = "0.34", default-features = false }
 
+[patch.crates-io]
+ibc        = { git = "https://github.com/cosmos/ibc-rs.git", revision = "b2d426f" }
+ibc-query  = { git = "https://github.com/cosmos/ibc-rs.git", revision = "b2d426f" }
```
Comment (rnbguy) [inline Cargo.toml]: `revision` was supposed to be [`rev`](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories). :smile:

File: Cargo.lock (hunk 1 above)
Comment (Farhad-Shabani) [reply Cargo.lock]: Whoops. I switch a lot between `main` and `feat/sov-support`, that's probably why!

File: Cargo.toml (hunk 2 above)
Comment (Farhad-Shabani) [reply Cargo.toml]: Oh, no 😄

**Review by rnbguy — APPROVED** [review]
LGTM :sparkles:

**Pull request merged**